src/test/java/pages/TutorialspointPage.java

```diff
@@ -0,0 +1,44 @@
+package pages;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class TutorialspointPage extends AbstractPage {
+
+    private static final String TUTORIALSPOINT_URL = "https://www.tutorialspoint.com/";
+
+    @FindBy(id = "gsc-i-id1")
+    private WebElement searchInputArea;
+
+    @FindBy(xpath = "//div[contains (text(), 'results')]")
+    private WebElement searchResults;
+
+    public TutorialspointPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public TutorialspointPage openPage() {
+        driver.get(TUTORIALSPOINT_URL);
+        driver.manage().window().maximize();
+        waitForPageLoad(driver);
+
+        return this;
+    }
+
+    public TutorialspointPage searchForPhrase() {
+        String searchValue = "submit() vs click()";
+        waitForPageLoad(driver);
+        searchInputArea.sendKeys(searchValue, Keys.ENTER);
+
+        return new TutorialspointPage(driver);
+    }
+
+    public int countTutorialspointSearchResults() {
+        waitForPageLoad(driver);
+        String searchResultsText = searchResults.getText();
+        return Integer.parseInt(searchResultsText.replaceAll("[\\D]", ""));
```

Comment:
add empty line before return statement

Reply:
Thank you for pointing, please see in the following commit.